src/cms/views/language_tree/language_tree_actions.py

```diff
@@ -67,3 +68,43 @@ def move_language_tree_node(
         logger.exception(e)
 
     return redirect("language_tree", **{"region_slug": region_slug})
+
+
+def delete_language_tree_node(request, region_slug, language_tree_node_id):
```

Comment:
```suggestion
@login_required
@region_permission_required
@permission_required("cms.manage_language_tree", raise_exception=True)
def delete_language_tree_node(request, region_slug, language_tree_node_id):
```
Please check the required permissions for the view, otherwise anonymous users, which are not logged in, can delete arbitrary language tree nodes.